docs/recipes/babelrc.md

```diff
@@ -0,0 +1,87 @@
+# Configuring Babel
+
+There are multiple options for configuring how AVA transpiles your tests using Babel.
+
+ - [Extend existing `.babelrc` without modification](#extend-existing-babelrc-without-modification)
+ - [Extend existing `.babelrc` with additional plugins or presets](#extend-existing-babelrc-with-additional-plugins-or-presets)
+ - [Extend an alternate config file (i.e. not `.babelrc`)](#extend-alternate-config-file)
+ - [Specify a complete config in `package.json`](#specify-a-complete-config-in-packagejson)
+ - [Notes](#notes)
+
+## Extend existing `.babelrc` without modification
+
+Use the `"inherit"` shortcut if you want your tests transpiled the same as your sources:
+
+`package.json`:
+
+```json
+{
+ "ava": {
+   "babel": "inherit"
+ }
+}
+```
+
+## Extend existing `.babelrc` with additional plugins or presets
+
+Set `babelrc` to `true`. This will use your `.babelrc` and extend it with any additional plugins specified.
+
+`package.json`:
+
+```json
+{
+  "ava": {
+    "babel": {
+      "babelrc": true,
+      "plugins": ["custom-plugin-name"],
+      "presets": ["custom-preset"]
+    }
+  }
+}
+```
+
+## Extend alternate config file.
+
+
+If, for some reason, you do not want to extend `.babelrc`, set the `extends` option to the alternate config you want to use during testing.
+
+`package.json`:
+
+```json
+{
+  "ava": {
+    "babel": {
+      "extends": "./babel-test-config.json",
+      "plugins": ["custom-plugin-name"],
+      "presets": ["custom-preset"]
+    }
+  }
+}
+```
+
+The above uses `babel-test-config.json` as the base config, and extends it with the custom plugins and presets specified.
+
+## Specify a complete config in `package.json`
+
+When the `babelrc` option is `false` (the default), `.babelrc` is not read for transpiling tests. You must specify your complete config in `package.json`.
+
+```json
+{
+  "ava": {
+    "babel": {
+      "babelrc": false,
```

Comment:
This is not really needed, and people will just copy-paste this. So I would leave it out.